docs/content/how-do-i-write-policies.md

```diff
@@ -307,13 +314,13 @@ collections of unique values. Just like other composite values, sets can be
 defined in terms of scalars, variables, references, and other composite values.
 For example:
 
-```ruby
-> s := {cube.width, cube.height, cube.depth}; n := count(s)
-+---+---------+
-| n |    s    |
-+---+---------+
-| 3 | [5,4,3] |
-+---+---------+
+```live:eg/cube/sets:module:merge_down
+s = {cube.width, cube.height, cube.depth}
```

Comment:
Same as previous comment. Looks like we're forcing things to be modules when they could just be queries.

Reply:
Done; changes the displayed output to a pretty table.